test/conformance/tests/test_sriov_operator.go

```diff
@@ -501,9 +501,12 @@ var _ = Describe("[sriov] operator", func() {
 				Expect(err).ToNot(HaveOccurred())
 
 				Eventually(func() bool {
-					stdout, stderr, err := pod.ExecCommand(clients, hostNetPod, "ip", "link", "show")
-					Expect(err).ToNot(HaveOccurred())
-					Expect(stderr).To(Equal(""))
+					var stdout, stderr string
+					Eventually(func(g Gomega) {
```

Comment:
nit: can you add a comment why this is needed ?

Reply:
sure done :)